edb/edgeql/compiler/inference/cardinality.py

```diff
@@ -332,6 +332,19 @@ def __infer_type_introspection(
     return ONE
 
 
+@_infer_cardinality.register
+def __infer_type_root(
+    ir: irast.TypeRoot,
```

Comment:
I know we have `root_rvar`, but do we think there is a better name than `root`?

Reply:
I'm not sure. I picked it by analogy with `root_rvar` but also don't love it.